src/libmongoc/tests/test-mongoc-ssl.c

```diff
@@ -205,7 +205,7 @@ static void
 test_non_existant_cafile (void)
 {
    mongoc_client_t *client = mongoc_client_new ("mongodb://localhost:27017/?tls=true&tlsCAFile=/nonexistant/ca.pem");
-   ASSERT (!mongoc_client_command_simple (client, "admin", tmp_bson ("{'ping': 1}"), NULL, NULL, NULL));
+   mongoc_client_command_simple (client, "admin", tmp_bson ("{'ping': 1}"), NULL, NULL, NULL);
```

Comment:
Should this fail even if the Windows has the cert file? If the `tlsCAFile` names a non-existent file, then do we really want to fall back to the Windows cert store? And if so, should it fail anyway since the user is explicitly requesting a non-existing CA cert?

Reply:
That may be a worthwhile later improvement. But at present, failure to load the CA [file logs](https://github.com/mongodb/mongo-c-driver/blob/023df6a377683c3b3621713b3901cc1f995651c9/src/libmongoc/src/mongoc/mongoc-secure-channel.c#L292) but does not propagate the error.

Reply:
Filed CDRIVER-5747 to propose propagating TLS set-up errors. To unblock the 1.28.1 release, I would rather keep the behavior as-is for now (no worse than the status quo).